packages/tizen_bundle/example/tizen/tizen-manifest.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.example.tizen_bundle_example" version="1.0.0" api-version="4.0" xmlns="http://tizen.org/ns/packages">
+    <profile name="mobile"/>
+    <ui-application appid="com.example.tizen_bundle_example" exec="Runner.dll" type="dotnet" multiple="false" nodisplay="false" taskmanage="true" api-version="4">
```

Comment:
https://github.com/flutter-tizen/flutter-tizen/wiki/New-plugin-submission-guideline#tizen-manifestxml

- Please change the profile name to `common`.
- The package and application IDs should start with `org.tizen.`.
- The `api-version="4"` attribute has been removed from the template and is no longer necessary. Please remove.

Reply:
Done. Thanks.